app.py

```diff
@@ -56,30 +56,28 @@ def email_redirect(path):
         account_repo.store_login(data)
         conn.commit()
 
-    # redirect for deprecated endpoint.
```

Comment:
Any idea why this is marked as removed? I don't think your code has much to do with unsubscribe...